server/models/notification.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  readunread: {
+    type: String,
+    required: true,
+  },
+  Date: {
+    type: Date,
```

Comment:
We have a date type but we're using enum strings, could this be a mistake? 

Suggestion: You can get rid of the 'Date' field and use mongoose [timestamps](https://masteringjs.io/tutorials/mongoose/timestamps)